src/layouts/partials/scripts.html

```diff
@@ -1,12 +1,11 @@
 {{ $aside := resources.Get "scripts/aside.js" }}
 {{ $base := resources.Get "scripts/base.js" }}
-{{ $gs_menu := resources.Get "scripts/gs_menu.js" }}
-{{ $js := slice $aside $base $gs_menu | resources.Concat "js/docs.js" | resources.Minify | fingerprint }}
+{{ $observer := resources.Get "scripts/IntersectionObserver.js" }}
+{{ $js := slice $aside $base $observer | resources.Concat "js/docs.js" | resources.Minify | fingerprint }}
```

Comment:
This adds the intersection observer JS file to our main JS, to avoid the separate HTTP request.